modules/bibdk_recommender/js/bibdk_recommender_covers.js

```diff
@@ -36,10 +36,11 @@
     var types = recommendation.closest('.js-slick-recommender').attr('data-recomole-types').split(",");
     BibdkRecommenderBehavior.filters_type = types;
     var recommendations = [];
-    elem.closest('.slick-track').find('.bibdk-recommender-cover-placeholder').each(function(index, element) {
-      recommendations.push($(this).attr('data-pid'));
+    elem.closest('.slick-track').find('.slick__slide').not('.slick-cloned').each(function(index, element) {
+      recommendations.push($(this).find('.bibdk-recommender-cover-placeholder').attr('data-pid'));
     });
-    BibdkRecommenderBehavior.result = unique(recommendations);
+    // BibdkRecommenderBehavior.result = unique(recommendations);
```

Comment:
Fjern udkommenteringen hvis det kode ikke er nødvendigt.

Reply:
og der er da også slettet noget kode ?? Er denher pull request up to date ?